src/3p-frame.js

```diff
@@ -117,8 +117,18 @@ export function getIframe(parentWindow, element, opt_type) {
   iframe.src = src;
   iframe.name = name;
   iframe.ampLocation = parseUrl(src);
-  iframe.width = attributes.width;
-  iframe.height = attributes.height;
+  // TODO(zhouyx): add check for width and height, delete later
+  // console.log('width', attributes.width);
+  // console.log('height', attributes.height);
+  if (attributes.hasOwnProperty('width')) {
+    // console.log('in')
+    iframe.width = attributes.width;
+  }
+  // TODO: <amp-ad-3p-impl> doesn't get width and height, what to do here?
+
+  if (attributes.hasOwnProperty('height')) {
+    iframe.height = attributes.height;
+  }
```

Comment:
I am thinking of getting width and height from element.parent(which is amp-ad) directly here?